packages/tables/resources/lang/en/table.php

```diff
@@ -16,17 +16,17 @@
 
     'fields' => [
 
-        'search_query' => [
-            'label' => 'Search',
-            'placeholder' => 'Search',
-        ],
-
         'bulk_select_page' => [
-            'label' => 'Select / Deselect all items for bulk actions.',
+            'label' => 'Select/Deselect all items for bulk actions.',
```

Comment:
```suggestion
            'label' => 'Select / deselect all items for bulk actions.',
```

Reply:
The space should not be there.

Reply:
hmm why?

Reply:
https://www.grammarly.com/blog/slash/

"When a slash signifies alternatives between only two words, don’t use spaces before or after."